src/main/java/org/mycore/mets/model/files/File.java

```diff
@@ -123,13 +125,13 @@ public String toString() {
 
     /*
      * (non-Javadoc)
-     * 
+     *
      * @see org.mycore.mets.model.IMetsElement#asElement()
      */
     public Element asElement() {
         Element file = new Element("file", IMetsElement.METS);
         file.setAttribute("ID", this.getId());
-        file.setAttribute("MIMETYPE", this.getMimeType());
+        file.setAttribute("MIMETYPE", Optional.ofNullable(getMimeType()).orElse("application/octet-stream"));
```

Comment:
I would not set the default "application/octet-stream" and not add the attribute at all if the mimetype is null